components/RiskFeed/Events/index.tsx

```diff
@@ -359,60 +393,69 @@ function EventSummary({ tx }: EventProps) {
-        <Box>
-          {data.detailedExecutionInfo.confirmations.map((confirmation) => (
-            <Value key={confirmation.signature}>
-              <Link href={address(confirmation.signer.value)} target="_blank" rel="noopener noreferrer">
-                {format(confirmation.signer.value)}
+    <>
+      <Box display="flex" flexDirection="column">
+        {data.txHash && (
+          <Row title={t('Transaction Hash')} inner={inner}>
+            <Value>
+              <Link href={transaction(data.txHash)} target="_blank" rel="noopener noreferrer">
+                {formatTx(data.txHash)}
               </Link>
             </Value>
-          ))}
-        </Box>
-      </Row>
-      {data.detailedExecutionInfo.executor && (
-        <Row title={t('Executor')}>
-          <Value>
-            <Link href={address(data.detailedExecutionInfo.executor.value)} target="_blank" rel="noopener noreferrer">
-              {format(data.detailedExecutionInfo.executor.value)}
-            </Link>
-          </Value>
+          </Row>
+        )}
+        <Row title={t('Submited At')} inner={inner}>
+          <Value>{parseTimestamp(data.detailedExecutionInfo.submittedAt / 1000, 'YYYY-MM-DD HH:mm:ss')}</Value>
         </Row>
-      )}
-      {isMultiSend ? (
-        data.txData.dataDecoded?.parameters?.[0]?.valueDecoded?.map((v, i) => (
-          <Row key={i} title={`${t('Action')} #${i + 1}`}>
-            <Decode to={v.to} data={v.dataDecoded} />
+        {data.executedAt && (
+          <Row title={t('Executed At')} inner={inner}>
+            <Value>{parseTimestamp(data.executedAt / 1000, 'YYYY-MM-DD HH:mm:ss')}</Value>
           </Row>
-        ))
-      ) : (
-        <Row title={t('Action')}>
-          <Decode to={data.txData.to.value} data={data.txData.dataDecoded} />
+        )}
+        <Row title={t('Signers')} inner={inner}>
+          <Box>
+            {data.detailedExecutionInfo.confirmations.map((confirmation) => (
+              <Value key={confirmation.signature}>
+                <Link href={address(confirmation.signer.value)} target="_blank" rel="noopener noreferrer">
+                  {format(confirmation.signer.value)}
+                </Link>
+              </Value>
+            ))}
+          </Box>
         </Row>
-      )}
-    </Box>
+        {data.detailedExecutionInfo.executor && (
+          <Row title={t('Executor')} inner={inner}>
+            <Value>
+              <Link href={address(data.detailedExecutionInfo.executor.value)} target="_blank" rel="noopener noreferrer">
+                {format(data.detailedExecutionInfo.executor.value)}
+              </Link>
+            </Value>
+          </Row>
+        )}
+        {isMultiSend ? (
+          <Row title={t('Actions')} inner={inner}>
+            <Box display="flex" flexDirection="column" gap={2}>
+              {data.txData.dataDecoded?.parameters?.[0]?.valueDecoded?.map((v, i) => (
+                <Decode key={i} to={v.to} data={v.dataDecoded} />
```

Comment:
avoid using array index as `key`. 
https://legacy.reactjs.org/docs/lists-and-keys.html
https://robinpokorny.com/blog/index-as-a-key-is-an-anti-pattern/